portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,36 +23,70 @@
 import java.util.List;
 import com.google.gson.Gson;
 import java.util.Arrays;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import java.util.*;
+import java.text.*;
+import java.sql.Timestamp;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;  
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    private ArrayList<String> messages;
-
-    @Override
-    public void init() {
-        messages= new ArrayList<String>();
-    }
-
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        //Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+Query query = new Query("Task");
```

Comment:
The formatting for several of the statements is still off.

Reply:
It should be updated now in the new commit.